### PR TITLE
Prevents the Maya submitter from crashing if a package in the resourc…

### PR DESCRIPTION
**Pull request by jesseconductor**
…es files is not lot available.

A little fix.. mostly for working in Dev.

### CODE REVIEW

File: conductor/submitter_maya.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 except ImportError, e:
     sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
+reload(submitter)
```
Comment (hoolymama) [inline conductor/submitter_maya.py:27]: Would be great to have a comment to say why this is needed.

**Review by hoolymama — APPROVED** [review]

File: conductor/submitter_maya.py (hunk 1 above)
Comment (jesseconductor) [reply conductor/submitter_maya.py]: @hoolymama Oops.. I had committed the wrong file. I pushed the correct changes.

File: conductor/submitter.py [hunk 2]
```diff
@@ -1445,7 +1445,15 @@ def getSelectedAvailablePackages(self):
...
-        return self.software_packages[package_id]
+        # This can happen when the local resources files is out of sync with active
+	# packages on the back-end
+        try:
+            software_package = self.software_packages[package_id]
+        except KeyError:
+            logger.warning("Unable to locate the package {} in the list of packages.".format(package_id))
+            return None
+        
+        return software_package
```
Comment (hoolymama) [inline conductor/submitter.py:1456]: If it's only about a KeyError, could you not do it with `get` at the place where this function is called?

`software_packages.get(package_id)`

just a nit, so I approved anyway.

**Review by hoolymama — APPROVED** [review]

**Review by lawschlosser — COMMENTED** [review]
btw the reason this has never been done in the past is because it's strictly a dev issue.  Adding code complexity in order to handle dev-only scenarios isn't the best idea IMO  (though it _is_ admittedly frustrating to deal with).  I'm not adamantly opposed to it, but typically I just comment out the offending entries in the resources.yaml file when doing dev work.  

Do as you see fit

File: conductor/submitter.py (hunk 2 above)
Comment (jesseconductor) [reply conductor/submitter.py:1456]: It could be but I thought it would be helpful to have the warning message.

**Comment by jesseconductor** [comment]
> btw the reason this has never been done in the past is because it's strictly a dev issue. Adding code complexity in order to handle dev-only scenarios isn't the best idea IMO (though it _is_ admittedly frustrating to deal with). I'm not adamantly opposed to it, but typically I just comment out the offending entries in the resources.yaml file when doing dev work.
> 
> Do as you see fit

Agreed that it's most likely to happen in dev (it could happen in prod too).. I got tired of making the changes over and over and it ultimately does handle the possible exception in a cleaner way - regardless of what scenario it might happen

**Pull request merged**